jdaviz/core/region_translators.py

```diff
@@ -16,7 +17,78 @@
                      EllipseAnnulusPixelRegion, EllipseAnnulusSkyRegion,
                      RectangleAnnulusPixelRegion, RectangleAnnulusSkyRegion, PixCoord)
 
-__all__ = ['regions2aperture', 'aperture2regions']
+__all__ = ['regions2roi', 'regions2aperture', 'aperture2regions']
+
+
+def regions2roi(region_shape, wcs=None):
+    """Convert a given ``regions`` shape to ``glue`` ROI.
+
+    This is the opposite of what is offered by
+    ``glue_astronomy.translators.regions.AstropyRegionsHandler.to_object``
+    but does not cover all the same shapes exactly.

```

Comment:
should this belong in `AstropyRegionsHandler.to_data` then?

Reply:
Theoretically, yes, but if you look at https://github.com/glue-viz/glue-astronomy/blob/main/glue_astronomy/translators/regions.py at the time of posting:

(a) `to_data` does not exist at all, so to implement one would include all the shapes that I do not care about as well, which is out of scope;
(b) getting anything into glue-astronomy takes too long and since this blocks Cubeviz feature, I want to avoid that overhead;
(c) I would also need to add more things to its `to_object` because it does not support all the shapes I want here (especially sky regions).

Reply:
(d) If I could help it, I would say we get rid of glue-astronomy altogether but that is a fiery debate for another day.

Reply:
ok, then let's leave it here for now (I can now definitely appreciate point b).  As for points a and c, I think something in glue-astronomy is better than nothing and we could always raise `NotImplementedError` for things we don't need yet, but we can migrate this over if/when there is a need.

Reply:
Can always open a follow-up issue to port the code there. I'll add that as post-merge todo.